app/src/main/res/layout/item_shop_management.xml

```diff
@@ -1,49 +1,62 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<layout
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:orientation="horizontal"
-    android:padding="@dimen/dp_3"
     >
-    <!--//todo image shop-->
-    <ImageView
-        android:layout_width="@dimen/dp_100"
-        android:layout_height="@dimen/dp_80"
-        android:layout_margin="@dimen/dp_2"
-        android:contentDescription="@string/shop"
-        />
-    <RelativeLayout
+
+    <data>
+
+        <variable
+            name="viewModel"
+            type="com.framgia.forder.screen.mainpage.shop.ItemShopViewModel"
+            />
+
+    </data>
+    <LinearLayout
         android:layout_width="match_parent"
-        android:layout_height="@dimen/dp_80"
-        android:layout_marginStart="@dimen/dp_5"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        android:padding="@dimen/dp_3"
         >
-        <TextView
-            android:id="@+id/text_shop_name"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_marginTop="@dimen/dp_10"
-            android:ems="8"
-            android:textSize="@dimen/sp_20"
-            />
-        <TextView
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_alignParentLeft="true"
-            android:layout_alignParentStart="true"
-            android:layout_below="@id/text_shop_name"
-            android:layout_marginTop="@dimen/dp_20"
-            android:ems="10"
-            android:textSize="@dimen/sp_15"
-            />
+        <!--//todo image shop-->
         <ImageView
-            android:layout_width="@dimen/dp_50"
-            android:layout_height="@dimen/dp_60"
-            android:layout_alignParentEnd="true"
-            android:layout_alignParentRight="true"
-            android:layout_alignTop="@+id/text_shop_name"
-            android:contentDescription="@string/shop_management"
-            app:srcCompat="@drawable/ic_next_fragment"
+            android:layout_width="@dimen/dp_100"
+            android:layout_height="@dimen/dp_80"
+            android:layout_margin="@dimen/dp_2"
+            android:contentDescription="@string/shop"
             />
-    </RelativeLayout>
-</LinearLayout>
+        <RelativeLayout
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/dp_80"
+            android:layout_marginStart="@dimen/dp_5"
```

Comment:
ERROR: [RtlCompat] To support older versions than API 17 (project specifies 14) you should *also* add `android:layout_marginLeft="@dimen/dp_5"`
 API 17 adds a `textAlignment` attribute to specify text alignment. However, if you are supporting older versions than API 17, you must *also* specify a gravity or layout_gravity attribute, since older platforms will ignore the `textAlignment` attribute.